web/lib/ozoneframework/php/core/Form.php

```diff
@@ -176,7 +176,8 @@ public function validate($fieldName = null){
 			if($this->validatorName == null) {
 				$this -> validatorName = "BaseFormValidator";
 			}
-
+			// As far as I can tell there's only the BaseFormValidator but I might be wrong.
+            $this->validatorName = "Ozone\Framework\\".$this->validatorName;
```

Comment:
Is `BaseFormValidator` something essential we should know about / keep an eye on?

Reply:
This looks like it's only relevant with regards to Data Forms. We may have to explore what validation, if any, is missing. But I'd be far more likely to let Laravel's robust validation engine do anything else we need.

Reply:
Sounds good to me. And data forms, while they should be carried over, are not high on my list of priorities.